<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Machine Helper</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="destinationInfoLabel">
        <property name="text">
         <string>Destinations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="DestinationsWidget" name="destinationsTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kind</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mount Point</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Usage</string>
         </property>
         <property name="toolTip">
          <string>Percent usage of the mounted destination.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Capacity</string>
         </property>
         <property name="toolTip">
          <string>Capacity of the volume.</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Free</string>
         </property>
         <property name="toolTip">
          <string>Free space within the volume.</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Backup Status</string>
         </property>
         <property name="toolTip">
          <string>Backup phase &amp; more when backup running</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Machines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="machinesTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>UUID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Model</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Destinations</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Backups</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fileSystemInfoLabel">
        <property name="toolTip">
         <string>File system info for Time Machine associated paths.</string>
        </property>
        <property name="text">
         <string>File System Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="mountPointsWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>441</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
         <property name="toolTip">
          <string>Path from mount point shown in Destination Info.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Backups</string>
         </property>
         <property name="toolTip">
          <string>Count of backups in a &quot;machine directory&quot;.</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Snap Type</string>
         </property>
         <property name="toolTip">
          <string>Snapshot type for &quot;backup&quot; directories.</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time elapsed between start &amp;amp; finish of backup in hh:mm:ss format.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Copied Size</string>
         </property>
         <property name="toolTip">
          <string>Amount of data copied in megabytes.</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Filesystem Type</string>
         </property>
         <property name="toolTip">
          <string>File system type name.</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Vol. Used</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="availableActionsLabel">
        <property name="toolTip">
         <string>Path based actions. Note that these actions are only available when one or more paths are selected.</string>
        </property>
        <property name="text">
         <string>Path Actions</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="availableActionsLayout">
        <item>
         <widget class="QPushButton" name="deletingPushButton">
          <property name="toolTip">
           <string>Administrative action: deletes selected backups!</string>
          </property>
          <property name="text">
           <string>Delete Backups...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uniqueSizePushButton">
          <property name="toolTip">
           <string>Unique size of paths within a machine directory.</string>
          </property>
          <property name="text">
           <string>Unique Size...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="verifyingPushButton">
          <property name="toolTip">
           <string>Verifies selected backups.</string>
          </property>
          <property name="text">
           <string>Verify Backups...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="restoringPushButton">
          <property name="toolTip">
           <string>Restores selected paths to a chosen path.</string>
          </property>
          <property name="text">
           <string>Restore...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DestinationsWidget</class>
   <extends>QTableWidget</extends>
   <header>destinationswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
